src/renderer/pages/modelling-ui-poc/column-select/column-select.tsx

```diff
@@ -0,0 +1,201 @@
+      hoverPointer();
+      return null;
+    }
+
+    const renderer = new WebGLRenderer({ alpha: true });
+    renderer.setSize(
+      webglContainerEl.clientWidth,
+      webglContainerEl.clientHeight
+    );
+
+    webglContainerEl.appendChild(renderer.domElement);
+
+    const cssContainerEl = document.querySelector('.css-wrapper');
+    invariant(cssContainerEl, 'could not find container tag');
+
+    const cssRenderer = new CSS3DRenderer();
+    cssRenderer.setSize(
+      cssContainerEl.clientWidth,
+      cssContainerEl.clientHeight
+    );
+
+    cssContainerEl.appendChild(cssRenderer.domElement);
+
+    const controls = new OrbitControls(camera, renderer.domElement);
+    const cssControls = new OrbitControls(camera, cssRenderer.domElement);
+
+    controls.enableRotate = false;
+    cssControls.enableRotate = false;
+
+    camera.position.set(0, 0, 10);
+    controls.update();
+    cssControls.update();
+
+    scene.background = new Color(COLORS.GRAY_DARK);
+
+    const grid = new InfiniteGrid({
+      spacingX: 10,
+      spacingY: 10,
+      color: new Color(COLORS.GRAY_SOFT),
+      distance: 2000,
+    });
+
+    grid.position.set(0, 0, -0.0001); // little tweak to make sure the grid stays behind all objects
+
+    // scene.add(grid);
+
+    const modelGrid = new ModelGrid({
+      disposition: {
+        x: [
+          {
+            label: 'A',
+
+            coordinate: 0,
+          },
+          {
+            label: 'B',
+            coordinate: 12.5,
+          },
+          {
+            label: 'C',
+            coordinate: 17.5,
+          },
+        ],
+        y: [
+          {
+            label: '1',
+            coordinate: 0,
+          },
+          {
+            label: '2',
+            coordinate: 7.5,
+          },
+          {
+            label: '3',
+            coordinate: 15,
+          },
+        ],
+      },
+      containerEl: cssContainerEl,
+    });
+
+    const columns = [
+      {
+        width: 2,
+        height: 2,
+        coorX: 15,
+        coorY: 17.5,
+        color: COLORS.NOTICE_SOFT_yellow,
+      },
+      {
+        width: 2,
+        height: 2,
+        coorX: 0,
+        coorY: 5,
+        color: COLORS.NOTICE_RED,
+      },
+      {
+        width: 1,
+        height: 3,
+        coorX: 0,
+        coorY: 10,
+        color: COLORS.NOTICE_RED,
+      },
+      {
+        width: 2,
+        height: 4,
+        coorX: 5,
+        coorY: 0,
+        color: COLORS.NOTICE_RED,
+      },
+      {
+        width: 2,
+        height: 1,
+        coorX: 10,
+        coorY: 0,
+        color: COLORS.NOTICE_RED,
+      },
+    ];
+    scene.add(modelGrid.object3D);
+    const columnGroup = ColumnsGroup({ columns });
+    scene.add(columnGroup);
+
+    window.addEventListener('click', onPointerMove, false);
+    function animate() {
+      requestAnimationFrame(animate);
+      controls.update();
+      // cssControls.update();
+      // cssRenderer.render(scene, camera);
```

Comment:
this PoC would still work if we uncomment this, right?

Reply:
I would say yes, but I'll check it

Reply:
it works :), just tested it